src/main/java/pl/touk/service/dto/NewDriverDto.java

```diff
@@ -0,0 +1,17 @@
+package pl.touk.service.dto;
+
+import lombok.*;
+
+import java.util.Date;
+
+@Setter
```

Comment:
dlaczego nie użyłeś adnotacji @Data z lomboka?

Reply:
Bo jej nie znałem. To jest pierwszy projekt w którym używam Lomboka.